manifests/nodejs.yml

```diff
@@ -92,10 +99,10 @@ tests/:
         Test_API_Security_Sampling_With_Delay: *ref_5_27_0
       test_custom_data_classification.py:
         Test_API_Security_Custom_Data_Classification_Capabilities: missing_feature
-        Test_API_Security_Custom_Data_Classification_Multiple_Scanners: missing_feature
-        Test_API_Security_Custom_Data_Classification_Negative: missing_feature
-        Test_API_Security_Custom_Data_Classification_Processor_Override: missing_feature
-        Test_API_Security_Custom_Data_Classification_Scanner: missing_feature
+        Test_API_Security_Custom_Data_Classification_Multiple_Scanners: v6.0.0-pre  # auto activation: might not be the earliest working version
+        Test_API_Security_Custom_Data_Classification_Negative: v6.0.0-pre  # auto activation: might not be the earliest working version
+        Test_API_Security_Custom_Data_Classification_Processor_Override: v6.0.0-pre  # auto activation: might not be the earliest working version
+        Test_API_Security_Custom_Data_Classification_Scanner: v6.0.0-pre  # auto activation: might not be the earliest working version
```

Comment:
@nccatoni hahaha you fell into the trap, `v6.0.0-pre` doesn't actually exists. It's the version we have in our master branch, but we never release it. Currently the latest version is `5.72.0`, thus these tests won't run correctly.

Also your parser replace inline comments by comments above, but here you're adding inline comments, which mean they'll be reformated at the next pass :laughing: